django_etebase/app.py

```diff
@@ -0,0 +1,27 @@
+import os
+
+from django.core.wsgi import get_wsgi_application
```

Comment:
It probably doesn't matter, but maybe it should be:
```
from django.core.asgi import get_asgi_application

os.environ.setdefault("DJANGO_SETTINGS_MODULE", "etebase_server.settings")
application = get_asgi_application()
```
Just because it's an asgi application (though I don't think there'll be a real difference).

Reply:
ill try to change that, i followed this [https://www.stavros.io/posts/fastapi-with-django/](url) , we are just doing this to get the db running , hopefully if we use sqlalchemy  it wont even be there :)